### DISCUSSION TITLE
How to use sync()

### DISCUSSION THREAD

**Discussion by 00Her0** [nesbox/TIC-80#1407] [Q&A] [answered]
I read that you can use `sync()` to save stuff like `mset()` to the (I mean that those changes could be seen in the map editor and not just while the cart was running), but I can't quite figure out how to do that. Anyone got an answer?

**Comment by joshgoebel** [comment, marked as answer]
Did you read the `sync` API on the wiki and look at the examples?

**Reply by 00Her0** [reply]
I figured it out! `sync()` has to be called like this: `sync(0,0,true)`. I read the wiki before, but I must have missed that.

**Reply by joshgoebel** [reply]
If the wiki can be improved or the wording more helpful please let us know.
